.circleci/config.yml

```diff
@@ -18,7 +18,7 @@ workflows:
     jobs:
       - linux-build
       - linux-build-options
-      - linux-storage-adapters-build
+      - linux-adapters-build
```

Comment:
Looks good. I wonder if we could use a more specific name, e.g. linux-parquet-s3-build? adapters term is way too generic IMO.

Reply:
made this change.